game/md2.ksy

```diff
@@ -0,0 +1,194 @@
+meta:
+  id: md2
```

Comment:
We already have [`game/quake_mdl.ksy`](https://github.com/kaitai-io/kaitai_struct_formats/blob/8b22d829/game/quake_mdl.ksy) and [`game/quake_pak.ksy`](https://github.com/kaitai-io/kaitai_struct_formats/blob/8b22d829/game/quake_pak.ksy) in the format gallery, so this should probably start with `quake_` (i.e. `quake_md2`) as well.

Reply:
Renamed to quake2_md2, hope that's fine